OWNERS_ALIASES

```diff
@@ -505,7 +503,6 @@ aliases:
     - thockin         # Network
     - timothysc       # Cluster Lifecycle, Scheduling
     - wojtek-t        # Scalability
-    - zehicle         # Cluster Ops
```

Comment:
Not currently an org member
ref: https://cs.k8s.io/?q=zehicle&i=fosho&files=&repos=kubernetes/org